wagtailio/project_styleguide/templates/patterns/components/streamfields/icon_bullets_list_block.html

```diff
@@ -0,0 +1,5 @@
+{% for icon_bullet in value.icon_bullets %}
+    <div>
```

Comment:
@engineervix this should be an `<ul>`
@chris-lawton what should this be? or are you guys going to follow up on this?
for context, this is the template that includes #188

Reply:
Should be a `<ul>` - the updated template including the FE is here: https://github.com/wagtail/wagtail.org/blob/next/wagtailio/project_styleguide/templates/patterns/components/streamfields/icon_bullets_list_block/icon_bullets_list_block.html